vncviewer/OptionsDialog.cxx

```diff
@@ -58,7 +58,7 @@ std::map<OptionsCallback*, void*> OptionsDialog::callbacks;
 static std::set<OptionsDialog *> instances;
 
 OptionsDialog::OptionsDialog()
-  : Fl_Window(450, 460, _("VNC Viewer: Connection Options"))
+  : Fl_Window(450, 550, _("VNC Viewer: Connection Options"))
```

Comment:
I would prefer to not keep growing this dialog. I wonder if the new dialog from #1410 should be merged first to solve this.

Reply:
I'm just back at the computer after being AFK for a week.  I took a look at #1410 and that looks pretty nice.  Do you have a sense for when that may merge?  I'm happy to rework my changes to use the new dialog.

Reply:
#1410 has spiralled a bit out of control, but I can try to get the option window part merged so it doesn't block the work here.